resolwe_bio/processes/chip_seq/macs2.yml

```diff
@@ -421,6 +440,10 @@
         re-error "--broad can't be combined with --call-summits!"
       {% endif %}
 
+      {% if settings.shift and settings.format == "BAMPE" %}
+        re-error "Values other than 0 are not supported when the format is BAMPE."
```

Comment:
State which input parameter values are affected in the error msg,

Reply:
Yes, thanks